lib/utils.pm

```diff
@@ -1959,4 +1960,8 @@ sub assert_secureboot_status {
     die "Error: SecureBoot is $statestr" if $state xor $expected;
 }
 
+sub is_opensuse_performance {
```

Comment:
I don't understand. What is "opensuse performance"? There should be an according performance scenario for SLE as well. I don't think you need this distinction

Reply:
removed the function.